### PR TITLE
625 converter update product menu

### LINKED ISSUE #625: [Product] Converter Update for Product Menu

### PR DESCRIPTION
**Pull request by rgravitvl**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #625 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://625-converter-update-product-menu--danaher-ls-aem--hlxsites.hlx.page/

### CODE REVIEW

**Review by sharanagoudapatil — APPROVED** [review]
LGTM. Thanks.

File: tools/importer/transformers/util.js [hunk 1]
```diff
@@ -170,10 +170,16 @@ export const pdfembed = (embedEl, document) => {
 
 export const videoembed = (embedEl, document) => {
   const videoEl = embedEl?.querySelector('iframe');
+  embedEl.innerHTML = '';
   const anc = document.createElement('a');
-  anc.href = videoEl.getAttribute('src');
+  let href = videoEl.getAttribute('src');
+  if (!href.startsWith('https:') && href.includes('vidyard')) {
+    href = `https:${videoEl.getAttribute('src').replace('?', '')}`;
```
Comment (duynguyen) [inline tools/importer/transformers/util.js:177]: I understand that if there is a question mark at the end of the video url, it should be ignored.
what if the url has a query param? e.g. `//play.vidyard.com/WwWUddqbVhBk8jtmGQ6dHF.html?videoId=abcd1234`, is this a possible case?
a safer would be just cutting the `?` if it is present at the end.

File: tools/actions/convert/test/fixtures/product1-converted.html [hunk 2]
```diff
@@ -92,6 +92,35 @@ <h2>CloneSelect Imager FL</h2>
...
       <h2>Streamlined Workflow</h2>
+      <div class="product-menu">
+        <div>
+          <div>
+            <p>
+              <img src="https://author-dummy.adobeaemcloud.com/content/dam/danaher/system/icons/product-menu-chevron.png" alt="">
+            </p>
+            <p>Image</p>
+          </div>
+          <div><a href="/content/experience-fragments/danaher/us/en/site/productdata/cloneselect-imager/master/jcr:content.html"></a></div>
```
Comment (duynguyen) [inline tools/actions/convert/test/fixtures/product1-converted.html:103]: I guess those fragment links will be converted as well?
e.g. `/fragments/productdata/cloneselect-imager/master`

File: tools/importer/transformers/util.js (hunk 1 above)
Comment (rgravitvl) [reply tools/importer/transformers/util.js]: Done

File: tools/actions/convert/test/fixtures/product1-converted.html (hunk 2 above)
Comment (rgravitvl) [reply tools/actions/convert/test/fixtures/product1-converted.html:103]: Added the paths

**Review by duynguyen — APPROVED** [review]

**Pull request merged**